.github/workflows/build.yml

```
@@ -8,6 +8,8 @@ jobs:
       matrix:
         os:
           - 'ubuntu-latest'
+          - 'windows-latest'
+          - 'macos-latest'
```

While I appreciate the inclusiveness here, the windows wheels are actually false-positive results (un-useable).

Personally, I'm rather anit-Apple, so I can't vouch for anything that VM produces. Github CI runners using macos actually have usage limits because they can't be served from Microsoft Azure servers. I doubt you'll hit these limits on this repo, but its yet another way Apple milks it's own enthusiasts for 💰.
```suggestion
          # - 'windows-latest'
          # - 'macos-latest'
```